01-Login/templates/home.html

```diff
@@ -6,11 +6,11 @@
   <body>
     {% if session %}
     <h1>Welcome {{session.userinfo.name}}!</h1>
-    <p><a href="/logout">Logout</a></p>
+    <p><a href="/logout" id="qsLogoutBtn">Logout</a></p>
     <div><pre>{{pretty}}</pre></div>
     {% else %}
-    <h1>Welcome Guest</h1>
-    <p><a href="/login">Login</a></p>
+    <h1 id="profileDropDown">Welcome Guest</h1>
```

Comment:
This is a little tenuous, but the tests look for the logout button in a dropdown, so need to click `#profileDropDown` before clicking the logout button